docs-v2/content/en/docs/pipeline-stages/deployers/cloudrun.md

```diff
@@ -9,23 +9,143 @@ featureId: deploy.cloudrun
 
 
 ## Deploying applications to Cloud Run
-
-Skaffold can deploy services to Cloud Run. If this deployer is used, all provided manifests must be valid Cloud Run services, using the serving.knative.dev/v1 schema.
+Skaffold can deploy services and jobs to Cloud Run. If this deployer is used, all provided manifests must be valid Cloud Run services, using the `serving.knative.dev/v1` schema, or valid Cloud Run jobs.
 See the [Cloud Run YAML reference](https://cloud.google.com/run/docs/reference/yaml/v1) for supported fields.
 
-This deployer will use the [application default credentials](https://cloud.google.com/docs/authentication/production#automatically) to deploy.  You can configure this to use your user credentials by running `gcloud auth application-default login`.
-
-# Features
-- As of Skaffold `v2.1.0`, Skaffold's Cloud Run Deployer now supports log streaming from the deployed Cloud Run Service(s)/Job(s).  NOTE: this requires installing the following `gcloud components`: `alpha`, `beta`, `cloud-run-proxy`, and `log-streaming`
-- Supports deploying Cloud Run `Service` and `Job` objects
+### Environment setup
+In order to use this deployer you'll need to configure some tools first.
+
+The deployer uses `gcloud` CLI to perform its tasks, so be sure it is installed in your environment. It will use the [application default credentials](https://cloud.google.com/docs/authentication/production#automatically) to deploy.  You can configure this to use your user credentials by running:
+```bash
+gcloud auth application-default login
+```
+
+To enable [Log streaming]({{< relref "#log-streaming" >}}) and [Port forwarding]({{< relref "#port-forwarding" >}}) some extra components are needed from `gcloud`. To install them run the following comand in your terminal:
+```bash
+gcloud components install --quiet \
+    alpha \
+    beta \
+    log-streaming \
+    cloud-run-proxy
+```
+
+From the previous command, `alpha` and `log-streaming` components are needed for [Log streaming]({{< relref "#log-streaming" >}}), `beta` and `cloud-run-proxy` components are needed for [Port forwarding]({{< relref "#port-forwarding" >}}).
+
+## Features
+
+### Cloud Run Services and Jobs deployment
+With Skaffold you can deploy Cloud Run [Services](https://cloud.google.com/run/docs/overview/what-is-cloud-run#services) and [Jobs](https://cloud.google.com/run/docs/overview/what-is-cloud-run#jobs) just referencing them from the `skaffold.yaml` file. The following example ilustrates a project using the Cloud Run deployer:
+
+With the following project folder structure:
+```yaml
+resources/
+  cloud-run-service.yaml
+  cloud-run-job.yaml
+skaffold.yaml
+```
+
+`cloud-run-service.yaml` content:
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: cloud-run-service-name
```

Comment:
```suggestion
  name: cloud-run-service-name # this service will be created in Cloud Deploy via Skaffold
```

Reply:
Change applied with the following text: `this service will be created in Cloud Run via Skaffold`: Changed `Cloud Deploy` to `Cloud Run`. Also I added the same comment for the Job.